setup.py

```diff
@@ -58,25 +58,25 @@ def is_requirement(line):
 
 
 setup(
-    name='django-babel-underscore',
+    name='enmerkar-underscore',
     version=version,
     description='Implements a underscore extractor for django-babel.',
     long_description=read('README.rst') + u'\n\n' + read('HISTORY.rst'),
     author='Christopher Grebs',
     author_email='cg@webshox.org',
-    url='https://github.com/EnTeQuAk/django-babel-underscore',
+    url='https://github.com/Zegocover/enmerkar',
     packages=find_packages('src'),
     package_dir={'': 'src'},
     include_package_data=True,
     tests_require=load_requirements('requirements/test.in'),
     install_requires=load_requirements('requirements/base.in'),
     entry_points="""
     [babel.extractors]
-    underscore = django_babel_underscore:extract
+    underscore = enmerkar_underscore:extract
     """,
     zip_safe=False,
     license='BSD',
-    keywords='django-babel-underscore',
+    keywords='enmerkar-underscore',
     classifiers=[
         'Development Status :: 2 - Pre-Alpha',
```

Comment:
Maybe update the development status while we're at it?  We've been using this in production for years, so pre-alpha doesn't seem appropriate.